index.bs

```diff
@@ -750,15 +750,19 @@ dictionary XRReferenceSpaceOptions {
   XRStationaryReferenceSpaceSubtype subtype;
 };
 
-[SecureContext, Exposed=Window] interface XRReferenceSpace : XRSpace {
-  attribute XRRigidTransform originOffset;
+
+[SecureContext, Exposed=Window,
+ Constructor(XRReferenceSpace base, XRRigidTransform originOffset)]
+interface XRReferenceSpace : XRSpace {
+  readonly attribute XRReferenceSpace? base;
```

Comment:
`[SameObject]` on both in accordance with https://github.com/immersive-web/webxr/issues/588 ?

Reply:
In accordance with our move to remove attributes that reflect the object's creation parameters, this should probably be removed altogether.